src/main/resources/templates/index.html

```diff
@@ -27,12 +27,14 @@
 
 <body>
   <main>
-    <tamuheader title="Sage"></tamuheader>
+    <tl-header page-title="SAGE"></tl-header>
```

Comment:
This should not be on the footer branch.

Reply:
Removed tl-header component from the footer feature branch.